lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -178,19 +174,47 @@ final class AddComicViewController: UIViewController {
         finishButton.tintColor = LucienTheme.dark
     }
 
+    private func darkenTextFieldBottomBorder(textField: UITextField) {
+        guard let bottomBorderSubLayer = textField.layer.sublayers else { return }
+        let bottomBorder = bottomBorderSubLayer[0] as CALayer
```

Comment:
I see this `bottomBorderSubLayer[0]` in several places. I see why that's convenient. But it's another case where I wouldn't want to rely on UI elements that you don't explicitly manage for this information. I would recommend you subclass UITextField and any textfields that have this bottom border can have a property called `bottomBorderLayer` that you can guarantee exists and is what you think it is.

Additionally if you use those subclassed UITextFields everywhere you can outsource some of the repetitive setup you're doing to that subclass.